src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -107,14 +107,21 @@ export class ChallengesubmitComponent implements OnInit {
   apiCall: any;
 
   /**
-   * Submissions remaining for the selected phase
+   * Selected phase submission conditions
```

Comment:
For the participant, in the `phases` section, I can see the updated no. of allowed submissions but in the `submit` section, they are unchanged and = `100000`. Is it happening on your side as well?

Reply:
@lunayach it works fine by my side

Reply:
Yes, it worked, after reverting the latest commit.